File: bot/src/main/java/net/slimediamond/atom/data/keys/Keys.java
package net.slimediamond.atom.data.keys;

import net.slimediamond.atom.common.data.ResourceKey;
import net.slimediamond.data.Key;

public final class Keys {
    public static final Key<String> TEST = Key.of(ResourceKey.atom("test"), String.class);
}

File: bot/src/main/kotlin/net/slimediamond/atom/data/Keys.kt
package net.slimediamond.atom.data

import net.slimediamond.atom.Atom
import net.slimediamond.atom.irc.api.Server
import net.slimediamond.data.Key
import net.slimediamond.data.identification.ResourceKey
import net.slimediamond.data.value.Value

class Keys {

    val SERVER: Key<Value<Server>> = Key.from(ResourceKey.of(Atom.instance, "server"), Server::class.java)

}